### DISCUSSION TITLE
Error registering user

### DISCUSSION THREAD

**Discussion by sergiosilva-nsj** [Erudika/scoold#309] [Q&A] [answered]
I followed the instructions to config the Para server, and scoold server, but, when I try register the admin user, the scoold server prints it:

[ERROR] 403 - 403 - Can't authenticate user with app 'app:para' using provider 'password'. Reason: clients aren't allowed to access root app.

After, I tried with older versions (erudikaltd/scoold:1.47.2 and erudikaltd/para:v1.43.3), and with these versiosn, I can configure with success, but, the UpVote is not working in these old versions...

Can someone help me?

**Comment by albogdano** [comment, marked as answer]
You will have to create a new app called "Scoold" for use with Scoold. The root app "app:para" is used by Para itself for storing system objects. Each Scoold server usually has its own separate Para app.

**Reply by sergiosilva-nsj** [reply]
How can I create a new app? I'm starting Para through docker image, so I'm not using the Para CLI.

Thank you very much!

**Reply by albogdano** [reply]
You should use the Para CLI tool because it's the easiest way to do it. Alternatively, you can call the `GET /v1/_setup/scoold` API endpoint in Para.

**Reply by sergiosilva-nsj** [reply]
I tried, but I'm getting this error:

/usr/local/lib/node_modules/para-cli/para-cli.js:24
import updateNotifier from 'update-notifier';
       ^^^^^^^^^^^^^^

SyntaxError: Unexpected identifier
    at Module._compile (internal/modules/cjs/loader.js:723:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)

**Reply by albogdano** [reply]
You should update Node.js to the latest version.

**Reply by sergiosilva-nsj** [reply]
It works! Thank you very much!
